### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by lxpollitt**
Add contributor/user documentation header notice and tightened content in a few places to be more contributor focussed.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,44 +4,27 @@
...
 
-Project Calico provides
-
-- A simple, pure layer 3 networking approach with no overlays for networking
-  "workloads" such as VMs and containers.
-- A distributed firewall implementing rich and flexible network policy,
-  imposed at ingress/egress to each workload.
-
-For more information see [the Project Calico website](http://www.projectcalico.org/learn/).
+<blockquote>
```
Comment (caseydavenport) [inline README.md]: A bit weird to use these tags in a markdown file, I think.

How about this:

```markdown
> Note that the documentation in this repo is targeted at Calico contributors.
> ## Documentation for Calico can be found at [docs.projectcalico.org](http://docs.projectcalico.org)
```

Formats like this:

> Note that the documentation in this repo is targeted at Calico contributors.
> ## Documentation for Calico can be found at [docs.projectcalico.org](docs.projectcalico.org)

Comment (lxpollitt) [inline README.md]: HTML fragments in markdown is an officially supported thing.  The HTML fragment proposed gives a better formatted notice for our purposes I think.

Comment (caseydavenport) [inline README.md]: Certainly supported and works.  Happy to merge if that's the way you'd prefer, but I much prefer the formatting of the way I suggested (both the markdown and the final rendered result).

Doesn't really matter though.

**Comment by caseydavenport** [comment]
LGTM.

**Pull request merged**